Kube/PrepareNode/README.md

```diff
@@ -0,0 +1,29 @@
+Tasks performed for Ubuntu 18.04:
+
+1. Install Docker dependencies
+1. Install the Docker repository
+1. Install the latest Docker (daemons and CLI)
+1. Disable swap in /etc/fstab
+1. Install and load /etc/sysctl.d/30-ericom_shield_common.conf
+ with common Shield sysctl values
+1. Install a custom /etc/docker/daemon.json and enable Docker service in systemd
+1. Add the user specified by the -u/--user option to the group docker
+1. Install and disable firewalld
```

Comment:
Why do we install firewalld in ubuntu?

Reply:
This procedure is common to both CentOS and Ubuntu. Basically we install it to make sure it is disabled (usually it is already installed).